salt/api-gateway/init.sls

```diff
@@ -34,11 +34,8 @@ proxy:
         - template: jinja
         - require:
             - api-documentation
-
-    service.running:
-        - name: nginx
-        - require:
-            - file: proxy    
+        - watch_in:
+            - service: nginx-server-service
```

Comment:
there was a problem with kong on xenial taking a second or two to start up and adding all the api endpoints and such were failing. I tried a bunch of things before adding in the 2 second delay on the kong service. I thought it may have been the nginx auto-reloading when it detected new conf files but that didn't pan out.